modules/nix-bitcoin-services.nix

```diff
@@ -32,14 +32,14 @@ in
   # Allow any traffic
   allowAnyIP = { IPAddressAllow = "any"; };
 
-   enforceTor = mkOption {
-     type = types.bool;
-     default = false;
-     description = ''
-       "Whether to force Tor on a service by only allowing connections from and
-       to 127.0.0.1;";
-     '';
-   };
+  enforceTor = mkOption {
+    type = types.bool;
+    default = true;
```

Comment:
I'm unsure what consequences this could have. @jonasnick

Reply:
This is intended to be just a [refactor](0a0254c840730757548eb5343543ccadb30e09b0) and should not change any functionality (unless I overlooked something).

I prefer having sane defaults and not override them unless necessary, but of course a point can also be made that explicit is better than implicit.

Reply:
I agree with sane defaults. But sane defaults should be set in nix-bitcoin.nix. People using affected modules outside of nix-bitcoin will become _insane_ if the module behaves in such an unexpected way.